gobblin-metrics/src/main/java/gobblin/metrics/event/sla/SlaEventSubmitter.java

```diff
@@ -0,0 +1,129 @@
+package gobblin.metrics.event.sla;
+
+import java.util.Map;
+import java.util.Properties;
+
+import lombok.AccessLevel;
```

Comment:
Sure. Currently we have this page https://github.com/linkedin/gobblin/wiki/IDE-setup that describes lombok at a high level. It points to lombok's IDE setup page. Do we want to add more?